src/content/special-interest-groups/neuroimaging/index.md

```diff
@@ -5,22 +5,19 @@ groupName: Neuroimaging
 leads:
   - AlexVCaron: Alex Valcourt Caron
   - arnaudbore: Arnaud Boré
-
-pipelines:
```

Comment:
the schema for special interest group pages requires a `pipelines` entry, therefore just make it an empty array (sorry if I was unclear before)